ftw/simplelayout/tests/test_alias_block.py

```diff
@@ -27,6 +28,17 @@ def setUp(self):
 
         self.intids = getUtility(IIntIds)
 
+    def assert_html(self, page_html, aliasblock_html):
+        diff = difflib.unified_diff(
```

Comment:
According to https://docs.python.org/2.7/library/difflib.html#difflib.unified_diff, this function is for creating a diff between two lists. It seems as if you give it full strings. Wouldn't this then diff every single character to one another, meaning if one word is already different the diff would get quite big?

Reply:
Good point I'll have a look, I did work on my end when I changed something in the middle. But indeed the api suggest something else 🤔

Reply:
If it is a problem I think you could just split the string by newlines and it would be ok.

Reply:
@Nachtalb That was a really good call. I only tested some white spaces... and they worked. 

Before:
<img width="470" alt="Screen Shot 2020-06-09 at 11 41 07" src="https://user-images.githubusercontent.com/437933/84132939-c5baae80-aa46-11ea-8379-aac2855a1eaa.png">

After:
<img width="321" alt="Screen Shot 2020-06-09 at 11 43 35" src="https://user-images.githubusercontent.com/437933/84132968-cf441680-aa46-11ea-839e-23db646dfd64.png">

Reply:
🎉